src/lib/workouts/jumpingjack.ts

```diff
@@ -0,0 +1,62 @@
+import type { Pose } from '@tensorflow-models/pose-detection';
+import Workout, { DEFAULT_MOVENET_CONFIG } from '.';
+
+export default class JumpingJack extends Workout {
+	private isLegsTogether: boolean;
+	private jumpingJacks: number;
+	private iterations: number;
+
+	constructor() {
+		super();
+		this.isLegsTogether = true;
+		this.iterations = 0;
+		this.jumpingJacks = 0;
+	}
+	getModelConfigurations() {
```

Comment:
Could you explain why you did this instead of the default (or did you just copy+paste pushup code 💀)?